bpf_test.go

```diff
@@ -266,4 +292,47 @@ func TestModuleLoadELF(t *testing.T) {
 	checkCgroupProgs(t, b)
 	checkSocketFilters(t, b)
 	checkTracepointProgs(t, b)
+
+	// test 2: New Module, reading from file, with pinning custom config
+	mounted, err := bpffs.IsMounted()
+	if err != nil {
+		t.Fatalf("error checking if %q is mounted: %v", bpffs.BPFFSPath, err)
+	}
+	if !mounted {
+		if err := bpffs.Mount(); err != nil {
+			t.Skipf("error mounting bpf fs, skipping test: %v", err)
+		}
+	}
+
+	var secParams = map[string]elf.SectionParams{
+		"cgroup/skb": elf.SectionParams{
+			PerfRingBufferPageCount:   0,
+			SkipPerfMapInitialization: false,
+			CustomConfig: []elf.PinCustomConfig{
+				elf.PinCustomConfig{
+					Id:      3,
+					PinPath: filepath.Join("fw", hex.EncodeToString(util.MakeSha256([]byte("testgroup1")))),
```

Comment:
The only difference to `testgroup2` seems to be that `util.MakeSha256` and `hex.EncodeToString` is used, i.e. it doesn't seem to test a different code path and is redundant? Do I miss something?

Reply:
@schu I'll just remove the sha256 test, as @alban suggested. So we won't need to test twice at all.